tests/serial/map-if-necessary/three-solvers/mixed-substeps/ParallelExplicit.cpp

```diff
@@ -0,0 +1,31 @@
+#ifndef PRECICE_NO_MPI
+
+#include "../helper.hpp"
+#include "testing/Testing.hpp"
+
+#include <precice/precice.hpp>
+#include <vector>
+
+BOOST_AUTO_TEST_SUITE(Integration)
+BOOST_AUTO_TEST_SUITE(Serial)
+BOOST_AUTO_TEST_SUITE(MapIfNecessary)
+BOOST_AUTO_TEST_SUITE(ThreeSolvers)
+BOOST_AUTO_TEST_SUITE(MixedSubsteps)
+
+BOOST_AUTO_TEST_CASE(ParallelExplicit)
+{
+  PRECICE_TEST("A"_on(1_rank), "B"_on(1_rank), "C"_on(1_rank));
+
+  std::vector<int> readMappings{2, 2, 2};
+  std::vector<int> writeMappings{4, 4, 4};
```

Comment:
Why do we still have four write mappings? The substeps shouldn't be mapped either in an explicit coupling scheme, no?

Reply:
Probably explained in the PR description, right?